src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+# Donate Page
+#------------------------------------------------------------------------------
+
+donate_page:
+  title: Donate
+  description: >
+    ESLintは、継続的なメンテナンスと開発のための寄付金に依存しています
+    現在、BACKER_COUNTの企業・団体・個人が、毎月MONTHLY_DONATIONSを寄付してくれています。
+  hero:
+    cta:
+      text: スポンサーを見る
+      url: /sponsors
+  donation_options:
+    title: 寄付方法
+    subtitle: 以下のサイトを通じて寄付を受け付けています。
+    open_collective:
+      title: Open Collective
+      subtitle: 法人向けで一番人気のプラン
+      cta:
+        text: Open Collectiveを通じて寄付をします。
+      items:
+        - text: 501(c)(3)非営利団体に直接寄付をする
+        - text: 一時的な寄付と定期的な寄付
+        - text: クレジットカード、銀行振り込み、PayPalで支払います
+        - text: オープンソースファンドを設立し、複数のプロジェクトに寄付をします
+        - text: ほとんどの国で利用できます
+    github_sponsors:
+      title: GitHubを通じて寄付をします
+      subtitle: 高度な機能とレポート機能
+      cta:
+        text: Githubで寄付をする
+      items:
+        - text: GitHub billingで寄付をする
+        - text: 一時的な寄付と定期的な寄付
+        - text: クレジットカードでのお支払い、またはGitHubの既存の課金方法（請求書発行を含む）をご利用いただけます。
+        - text: GitHub プロフィールに "sponsor" バッジを表示する
+        - text: 既存のGitHubログインを使用できます
+  budget:
+    title: お金の使い道
+    subtitle: >
+      ESLintプロジェクトは、寄付されたお金をプロジェクトやESLintコミュニティ全体のために使っています。
+      一般的には、毎月下記のような分野にお金を使っています
+    items:
+      - name: チーム開発
+        text: >
+          チームメンバーには貢献期間の長さに応じて時給が支払われ、現在はTSCメンバーとレビュアーの時給はHOURLY_RATE_TSC USDとなっています。
+          コミッターの時給はHOURLY_RATE_COMMITERS USDとなっています.
+      - name: 貢献者プール
+        text: >
+          私たちは毎月CONTRIBUTOR_POOL_BUDGET USDを、プロジェクトに多大な貢献をした外部貢献者への支払いに充てています。
+          貢献には、コードの提出、ドキュメントの作成、Discordサーバでの質問への回答などが含まれます。このお金はTSCによって毎月配分されます。
+      - name: 依プロジェクト
+        text: >
+          私たちは毎月DEPENDENCIES_BUDGET USDを積み立て、ESLintが依存する他のプロジェクトをサポートするために役立てています。
+          私たちは、資金を得るのに十分な幸運に恵まれたオープンソースプロジェクトは、自分たちが依存している小規模な
+          プロジェクトを支援する責任があると考えます。
+      - name: コミュニティ・プロジェクト
+        text: >
+          私たちは毎月COMMUNITY_BUDGET USDを確保して、プラグインやパーサー、その他ESLintのエコシステム全体にとって有益なプロジェクト
+          をサポートするために役立てています。私たちは、プロジェクトの強さはそのエコシステムにあると信じており、
+          重要なプロジェクトが確実に維持され続けることを望んでいます。
+      - name: サポートシステム
+        text: >
+          Google Workspaceやクラウドストレージなど、チームがプロジェクトを管理するために使用するソフトウェアの費用として、
+          毎月少額を支払っています。
+    footer:
+      text: >
+        また、時々、この資金を使用して、請負業者に以下の作業を依頼することがあります：プロジェクトにとって重要で、チームには時間や専門知識がなくて
+        自分たちの手で完成できない。例えば、最近、このウェブサイトのリニューアルの為にデザイナーと開発者を雇いました。
+        そのようなプロジェクトのために貯蓄をすることができるよう、我々は毎月の支出を収入より少なくするよう心がけています。
+  donation_tiers:
+    title: 寄付のティアシステム
+    subtitle: 寄付の規模は問いませんが、レベルごとに異なる報酬を設定するティアシステムを導入しています。
+    cta:
+      open_collective_text: >
+        <span class="visually-hidden">Open Collective</span>で寄付をします
+      github_sponsors_text: >
+        <span class="visually-hidden">GitHub</span>で寄付をします
+    items:
+      - name: プラチナスポンサー
+        amount: $2000+
+        frequency: 毎月
+        text: 2時間のサポート、ホームページとReadmeへの最高レベルのロゴ掲載。その上に、Gold Tierの内容の全て。
```

This may be a more Japanese way of saying it.

```suggestion
        text: 2時間のサポート、ホームページとReadmeの最上段にロゴを掲載。さらに、Gold Tierの全ての内容を提供します。
```